kyo-bench/src/main/scala/kyo/bench/ForkJoinContentionBench.scala

```diff
@@ -37,7 +37,7 @@ class ForkJoinContentionBench extends Bench.ForkOnly[Unit]:
         val forkAllFibers     = ZIO.foreach(range)(_ => forkFiber)
         val forkJoinAllFibers = forkAllFibers.flatMap(fibers => ZIO.foreach(fibers)(_.await).unit)
 
-        ZIO.collectAllPar(Seq.fill(parallism)(forkJoinAllFibers)).unit
+        ZIO.collectAll(Seq.fill(parallism)(forkJoinAllFibers.forkDaemon)).flatMap(ZIO.foreach(_)(_.join)).unit
```

Comment:
I am a bit torn on this change. On one hand, it's best to be fair to all libraries and attempt to benchmark the true performance. However, this is a bit misleading to potential users. It's probable that a majority of ZIO code will directly use the `*Par` variants, thus this benchmark is not representative of that. Programmers will use the functions provided by the library.

Reply:
Not sure I agree. The main value of these benchmarks is to guide optimization efforts, hopefully even beyond Kyo. The semantics of the code is the same with this change and the execution seems also the same as the other targets. Removing the noise due to the current collectAllPar performance issue provides a better signal to measure the fork/join performance.

Reply:
I agree with that.